src/compiler/diagnosticMessages.json

```diff
@@ -4038,6 +4054,10 @@
         "category": "Error",
         "code": 5094
     },
+    "Option 'preserveValueImports' may be enabled only when 'module' is set to 'es2015' or later.": {

```

Comment:
The wording here is inconsistent with other similar messages, such as:

https://github.com/microsoft/TypeScript/blob/f8c686974525a4f7be9d0a5e17f2d63b1058ea0f/src/compiler/diagnosticMessages.json#L3877

https://github.com/microsoft/TypeScript/blob/f8c686974525a4f7be9d0a5e17f2d63b1058ea0f/src/compiler/diagnosticMessages.json#L3885

https://github.com/microsoft/TypeScript/blob/f8c686974525a4f7be9d0a5e17f2d63b1058ea0f/src/compiler/diagnosticMessages.json#L3881

https://github.com/microsoft/TypeScript/blob/f8c686974525a4f7be9d0a5e17f2d63b1058ea0f/src/compiler/diagnosticMessages.json#L3889

https://github.com/microsoft/TypeScript/blob/f8c686974525a4f7be9d0a5e17f2d63b1058ea0f/src/compiler/diagnosticMessages.json#L3893

Not that we're terribly consistent, but "cannot be specified" and "can only be used" are the most common qualifiers we use.

Reply:
I hate “can only be used” but will match existing options.

---

> > If you only see one movie this year...
> 
> Normal human beings frontload the word “only” at the beginning of a sentence. Copy editors will tend to pick up that “only” and drop it next to the thing that’s being “only”d:
> 
> > If you see only one movie this year...
> 
> Or, for instance:
> 
> > **NORMAL HUMAN BEING:** You can only watch a movie ironically so many times before you’re watching it earnestly.
> >
> > **COPY EDITOR:** You can watch a movie ironically only so many times before you’re watching it earnestly.
> 
> Does the latter perhaps sound a bit stilted? Perhaps it does, but to be perfectly honest, there’s a certain tautness in slightly stilted prose that I find almost viscerally thrilling.
> 
> I also think that readers don’t much notice when prose is wound up a bit too tight but may well, and not favorably, notice overloose prose.
> 
> Moreover, a loosely placed “only” can distort the meaning of a sentence entirely.
> 
> That said, in copyediting fiction, especially fiction with an informal narrative voice and, even more so, dialogue in fiction, I’m most likely to leave the “only” where the author set it.

— _Dreyer’s English_, p. 253